### PR TITLE
docs: webview approach

### LINKED ISSUE #6: WebView approach not working
## Whem I follow the guidelines for the webview approach, vendorId tag is not reflected.
Steps to reproduce:
1. Create view in swiftUI:
```swift
import SwiftUI
import WebKit
 
struct ContentView: View {
    var body: some View {
        Webview(url: URL(string: "https://eager-hermann-4ea017.netlify.app")!)
    }
}
 
struct Webview: UIViewRepresentable {
    let url: URL
 
    func makeUIView(context: UIViewRepresentableContext<Webview>) -> WKWebView {
        let webview = WKWebView()
 
        let vendorId = UIDevice.current.identifierForVendor?.uuidString ?? "undefined"

        let script = WKUserScript(source: "window.fingerprintjs.vendorId = \(vendorId)", injectionTime: .atDocumentStart, forMainFrameOnly: false)
 
        webview.configuration.userContentController.addUserScript(script)
 
        let request = URLRequest(url: self.url, cachePolicy: .returnCacheDataElseLoad)
        webview.load(request)
 
        return webview
    }
 
    func updateUIView(_ webview: WKWebView, context: UIViewRepresentableContext<Webview>) {
        let request = URLRequest(url: self.url, cachePolicy: .returnCacheDataElseLoad)
        webview.load(request)
    }
}
```

2. Create hmlt page with FPJS on the URL specified in the `Webview(url: URL(string: "https://eager-hermann-4ea017.netlify.app")!)`
``` html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta name="description" content="Webpage description goes here" />
    <meta charset="utf-8" />
    <title>Change_me</title>
  </head>
  <body>
    <div class="container"></div>
    <script>
      const vendorId =
        window.fingerprintjs && window.fingerprintjs.vendorId
          ? window.fingerprintjs.vendorId
          : 'not-in-ios-context'

      function initFingerprintJS() {
        // Initialize an agent at application startup.
        const fpPromise = FingerprintJS.load({
          token: 'tQUwQQOuG9TNwqc6F4I2',
          region: 'eu',
          endpoint: 'https://fp.martinmakarsky.com',
        })

        // Get the visitor identifier when you need it.
        fpPromise
          .then((fp) =>
            fp.get({
              tag: {
                deviceId: vendorId,
                deviceType: 'ios',
              },
              linkedId: 'makma',
              extendedResult: true,
            })
          )
          .then((result) => {
            console.log(result.visitorId);
            var el = document.createElement("h1");
            el.innerText = result.visitorId;
            document.body.appendChild(el);
          })
      }
    </script>
    <script
      async
      src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs-pro@3/dist/fp.min.js"
      onload="initFingerprintJS()"
    ></script>
  </body>
</html>
```
3. Run iOS app, get visitorId and check tag on server API
Expected output:
```json
...
            "linkedId": "makma",
            "timestamp": 1629209018949,
            "time": "2021-08-17T14:03:38Z",
            "url": "https://eager-hermann-4ea017.netlify.app/",
            "tag": {
                "deviceId": "vendor-id-from-ios-context",
                "deviceType": "ios"
            }
...
```
Actual output:
```json
...
            "linkedId": "makma",
            "timestamp": 1629209018949,
            "time": "2021-08-17T14:03:38Z",
            "url": "https://eager-hermann-4ea017.netlify.app/",
            "tag": {
                "deviceId": "not-in-ios-context",
                "deviceType": "ios"
            }
...
```

Note: I've also tried to set window object after `webview.load(request)`

Questions and suggestions:
- is webview approach documented well?
- is swift UI somehow specific?
- should it be `window.fingerprintjs.vendorId` or `window.FingerprintJS.vendorId`?
- I'd like to see full webview example in readme e.g. for SwiftUI

### PR DESCRIPTION
**Pull request by makma**
Fixes #6

### CODE REVIEW

**Review by spalt08 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ function initFingerprintJS() {
     .then((fp) =>
       fp.get({
         tag: {
-          deviceId: window.fingerprintjs.vendorId, // use vendor ID as device ID
+          deviceId: window.fingerprintjs, // use vendor ID as device ID
```
Comment (spalt08) [inline README.md:101]: `window.fingerprintjs.vendorId`

Comment (spalt08) [reply README.md]: fixed in 0464909

Comment (makma) [reply README.md]: No, it's not working this way.
Take a look: https://eager-hermann-4ea017.netlify.app/

Comment (spalt08) [reply README.md]: I think your example is quite different. I change the one above

Comment (spalt08) [reply README.md]: Since we are suggesting to declare the `window.fingerprintjs` as an object with the `vendorId` property

Comment (makma) [reply README.md]: sounds good, ty

**Pull request merged**